designer/core/src/main/java/com/percussion/client/models/impl/PSHierarchyManager.java

```diff
@@ -208,7 +215,7 @@ public IPSHierarchyNodeRef[] cloneChildren(
       }
       
       IPSHierarchyNodeRef[] validRefsArray = 
-         validRefs.toArray(new IPSHierarchyNodeRef[validRefs.size()]);
+         validRefs.toArray(new IPSHierarchyNodeRef[0]);
```

Comment:
It was setting it to validRefs.size() and now set to 0...

Reply:
It's a performance thing: https://stackoverflow.com/questions/53284214/toarray-with-pre-sized-array